postcss.config.js

```diff
@@ -0,0 +1,18 @@
+/** @type {import('postcss-load-config').Config} */
+module.exports = {
+  plugins: [
+    require('postcss-preset-env')({
+      stage: 2, // https://preset-env.cssdb.org/features/#stage-2
+      features: {
+        'nesting-rules': {noIsPseudoSelector: true},
+        'focus-visible-pseudo-class': false,
+        'logical-properties-and-values': false,
```

Comment:
Curious to hear why they are false 👀

Reply:
Took inspiration from https://github.com/primer/view_components/blob/main/postcss.config.js possibly @langermank can provide more insights

Reply:
I see @simurai/@jonrohan worked on this file and found some comments here: https://github.com/primer/view_components/pull/1638/files#r1032151244

Interesting stuff.

Reply:
In some of our cases, the output was putting more CSS than we needed for the browsers we supported. I think the focus-visible-pseudo-class was also adding a `.focus` class which was meant to be paired with JS. So it was on, until disabled.